doc/decisions/frontend_documentation.md

```diff
@@ -0,0 +1,99 @@
+  - It is not actively maintained anymore.
+  - It does not generate markdown files, so an additional step is needed, to get just the most basic information.
+
+- JSDoc: [JSDoc](https://jsdoc.app/) is an API documentation generator for JavaScript.
+It is more suited for JavaScript.
+
+## Decision
+We will use [TypeDoc](https://typedoc.org/) with the markdown plugin to extract documentation from the source code.
+
+## Rationale
+
+- TypeDoc has automatic support for most Tags of JSDoc via the TypeScript compiler.
+- Additional Tags are supported as well.
+- It is easy to configure and invoke and extensible via plugins.
+
+## Considered Alternatives (viewing components)
+
+- React Styleguidist: [React Styleguidist](https://react-styleguidist.js.org/)
+Is a popular and maintained solution to develop a project-wide style guide in React.
+It has fewer features than Storybook.
+It is hard to setup, as it assumes the usage of Create React App.
+- Docusaurus: [Docusaurus](https://docusaurus.io/)
+Is a popular solution for generating static documentation websites.
+It is more targeted towards manually written documentation and has less features specifically targeting how a component should be used.
+
+
+## Decision
+We will use [Storybook](https://storybook.js.org/) to enable a project wide component overview and style-guide.
+It has support for `MDX`, a format that enables embeddable components in markdown.
+It has support for automated testing via `Jest` and `Playwright`.
+
+## Rationale
+
+With Storybook the extracted documentation from TypeDoc can viewed as standalone documentation pages.
+It works well with Vite and is relatively easy to setup.
+
+## Implications
+
+- To install storybook a `.npmrc` file with `legacy-peer-deps=true` is required.
+This causes npm to behave like in version 6, not installing peer-dependencies automatically.
+This should not cause any problems down the line, but storybook could be extracted into it's own package, with more configuration involved.
+
+- A little plugin needs to be developed to link `Storybook` and the extracted markdown files from `TypeDoc` automatically.
+
+## Related Decisions
+
+## Notes
+
+### Notes about embedded markdown
+
+Without the plugin, we will need to write a `.mdx` file for each extracted markdown file.
+Also links inside the markdown will not work without the plugin.
+```jsx
+import Md from './docs/enums/bindings_definitions.Quality.md';
+import './markdown.css';
+import { Description, Meta } from '@storybook/addon-docs/blocks';
+import React from 'react';
+
+<Meta
+  title="Enumeration: Quality"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: { hidden: true },
+    },
+  }}
+/>
+
+<Md />
+```
+
+### How to structure a storybook
+https://storybook.js.org/blog/structuring-your-storybook/
```

Comment:
This should then also be in the main documentation.

Reply:
I added it to the Helpful Links section